package.json

```diff
@@ -87,7 +87,7 @@
     "qs": "^6.3.0",
     "react": "^16.9.0",
     "react-bootstrap": "^0.32.1",
-    "react-dom": "^16.9.0",
+    "react-dom": "^17.0.2",
```

Comment:
This is what is causing the tests to fail by the way -- something in the stop viewer is not compatible with React 17

Reply:
I've tested downgrading React locally and the menu still seems to work and the tests pass!

Reply:
What was the reason for trying to upgrade React?

Reply:
@binh-dam-ibigroup I think one of the plugin libraries required it and updated when I installed a sliding pane or a react submenu ( I forget which it was now). I will revert back.